src/containers/PrivacyPolicy/index.tsx

```diff
@@ -0,0 +1,341 @@
+        process your data. You maintain the right to withdraw such consent at any time. If you do not agree to our use
+        of your personal data in line with this Policy, please do not use our website.
+        <h2 className="PrivacyPolicy__subtitle">Collecting Your Personal Data</h2>
+        <p>We collect information about you in the following ways:</p>
+        <strong>Information You Give Us.</strong>
+        <br />
+        This includes:
+        <ul className="PrivacyPolicy__list">
+          <li>
+            the personal data you provide when you register to use our website, including your name, postal address,
+            email address, telephone number, username, password and demographic information (such as your gender)
+          </li>
+          <li>
+            the personal data that may be contained in any video, comment or other submission you upload or post to the
+            website
+          </li>
+          <li>
+            the personal data you provide in connection with our rewards program and other promotions we run on the
+            website
+          </li>
+          <li>
+            the personal data you provide when you report a problem with our website or when we provide you with
+            customer support
+          </li>
+          <li>the personal data you provide when you make a purchase thorough our website</li>
+          <li>the personal data you provide when you correspond with us by phone, email or otherwise</li>
+        </ul>
+        <p>
+          <strong>Information from Social Networking Websites.</strong>
+          <br />
+          Our website includes interfaces that allow you to connect with social networking sites (each a “SNS”). If you
+          connect to a SNS through our website, you authorize us to access, use and store the information that you
+          agreed the SNS could provide to us based on your settings on that SNS. We will access, use and store that
+          information in accordance with this Policy. You can revoke our access to the information you provide in this
+          way at any time by amending the appropriate settings from within your account settings on the applicable SNS.
+        </p>
+        <p>
+          <strong>Information Automatically Collected.</strong>
+          <br />
+          We automatically log information about you and your computer or mobile device when you access our website. For
+          example, when visiting our website, we log your computer or mobile device operating system name and version,
+          manufacturer and model, browser type, browser language, screen resolution, the website you visited before
+          browsing to our website, pages you viewed, how long you spent on a page, access times and information about
+          your use of and actions on our website. We collect this information about you using cookies.
```

Comment:
your use of => your use

We collect this information about you using cookies. => To collect this information about you, we use cookies.